modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb

```diff
@@ -187,7 +187,7 @@
         expect(response.body).to match_json_schema('VAOS_v2_appointments')
 
         uniq_statuses = response.parsed_body['data'].map { |appt| appt.dig('attributes', 'status') }.uniq
-        expect(uniq_statuses).to match_array(%w[BOOKED CANCELLED])
+        expect(uniq_statuses).to match_array(%w[BOOKED])
```

Comment:
I can only assume all the cancelled appointments were more than 30 days in the past.